docs/source/howto.rst

```diff
@@ -0,0 +1,98 @@
+.. _how-to:
+
+#############
+How-To Guides
+#############
+
+.. _how-to:install_archive:
+
+Installing from archive or folder
+=================================
+
+In case the automated install commands fail, or the pseudopotential family you want to use is not supported, you can install the pseudopotential family manually with the following command:
```

Comment:
This seems to refer implicitly to another location where the automated install commands are described. Is this the front page? I think there we only have the `install sssp` high-lighted correct? Should we add a first section here to mention/explain `install sssp` and `install pseudo-dojo`? Then this following section will make more sense